phylum_project/src/lib.rs

```diff
@@ -109,6 +110,7 @@ pub fn find_project_conf(
     // Convert to a canonicalized path so removing components navigates up the
     // directory hierarchy.
     let mut path = starting_directory.as_ref().canonicalize().ok()?;
+    path = dunce::simplified(&path).to_path_buf();
```

Comment:
One of the benefits of `dunce::simplified()` is that it's a no-op on non-Windows platforms... But when you add `.to_path_buf()`, it's a copy everywhere... That might be optimized away, but who knows...

You can just replace the call to `canonicalize` here instead of using `simplified`

```suggestion
    let mut path = dunce::canonicalize(starting_directory.as_ref()).ok()?;
```

Reply:
Good stuff. I knew I wouldn't get all this Rust stuff right...thanks!